include/tipsy/binary-to-float.h

```diff
@@ -78,13 +75,9 @@ union FloatBytes
         bytes[3] = (b3 & BIT_8_MASK) | EXPONENT_FILL;
     }
 
-    FloatBytes(unsigned char b1, unsigned char b2, unsigned char b3) noexcept
-    {
-        bytes[0] = b1;
-        bytes[1] = b2;
-        bytes[2] = b3 & LOW_7_MASK;
-        bytes[3] = (b3 & BIT_8_MASK) | EXPONENT_FILL;
-    }
+    constexpr FloatBytes(unsigned char b1, unsigned char b2, unsigned char b3) noexcept : bytes{b1, b2, (unsigned char)(b3&LOW_7_MASK),
```

Comment:
I assume this formatting is a result of clang-format?  I really dislike this wrap style for parameters (aligning with `(`), but I can live with it. In some codebases you end up with an absurd river of parameters way off to the right.  My preference is all wrapping of a line when necessary is a single indent.

I'd rather start the initializer list on a new line, starting with the `:` with a single indent. The initializer list is a separate thought that should be on a separate line: it's part of the implementation and not the parameters for the constructor. It's a preamble to the body.
That said, I'm not religious about formatting, and I'll only grumble now and then. like this comment, and now we can move on :-)

Reply:
I will just use the clang format file to auto format it. In surge land we decided that was good long ago so whatever it does is what I’ll merge! 

I will sweep that in and also add a build strange which fais a pr if not formatted. That’s by far the best solution I’ve found to talking about formatting in open source projects :)